python/hawkey/hawkeymodule.cpp

```diff
@@ -283,6 +283,8 @@ PYCOMP_MOD_INIT(_hawkey)
     PyModule_AddIntConstant(m, "PKG_URL", HY_PKG_URL);
     PyModule_AddIntConstant(m, "PKG_VERSION", HY_PKG_VERSION);
 
+    PyModule_AddIntConstant(m, "IGNORE_MODULAR_EXCLUDES", HY_IGNORE_MODULAR_EXCLUDES);
+    PyModule_AddIntConstant(m, "IGNORE_NON_MODULAR_EXCLUDES", HY_IGNORE_NON_MODULAR_EXCLUDES);
```

Comment:
You also need to add these to `python/hawkey/__init__.py`. And what about `HY_APPLY_EXCLUDES`? I know it's the default, but I think it should be a part of the API for completeness.